src/luarule.cc

```diff
@@ -87,45 +88,59 @@ void LuaRule::code (const std::string &newCode)
 int LuaRule::evaluate (const MetricList &metricList, PureAlert &pureAlert)
 {
     log_debug("LuaRule::evaluate %s", _name.c_str());
+    int res = 0;
```

Comment:
here we assume RULE_RESULT_UNKNOWN != 0

Reply:
No definition for result good (=0), should be added